src/app/technical-record-search/test-result.service.ts

```diff
@@ -34,7 +34,9 @@ export class TestResultService {
         `${this._apiServer.APITestResultServerUri}/test-results/${systemNumber}`,
```

Comment:
Backend equivalent ticket 14406:
AC2- Update testResult API PUT call

GIVEN I am the testResults API specification
WHEN I am updated into a new version with the PUT request
THEN the path parameter will be updated to contain test-results/testResultId

**Please update PUT call accordingly** ( in the service, effect, unit tests, etc.)